lib/src/cli.rs

```diff
@@ -183,6 +183,24 @@ pub(crate) enum InstallOpts {
     /// will be wiped, but the content of the existing root will otherwise be retained, and will
     /// need to be cleaned up if desired when rebooted into the new root.
     ToExistingRoot(crate::install::InstallToExistingRootOpts),
+    /// Intended for use in environments that are performing an ostree-based installation, not bootc.
+    ///
+    /// In this scenario the installation may be missing bootc specific features such as
+    /// kernel arguments, logically bound images and more. This command can be used to attempt
+    /// to reconcile. At the current time, the only tested environment is Anaconda using `ostreecontainer`
+    /// and it is recommended to avoid usage outside of that environment. Instead, ensure your
+    /// code is using `bootc install to-filesystem` from the start.
+    #[clap(hide = true)]
+    EnsureCompletion {
+        /// When provided, we assume that we're being invoked from our own
+        /// ostree-ext codebase.
+        #[clap(long)]
```

Comment:
```suggestion
        #[clap(long, requires = "stateroot")]
```

Reply:
FWIW it only worked to make them mutually required

```
diff --git a/lib/src/cli.rs b/lib/src/cli.rs
index acd986da..572fd0d2 100644
--- a/lib/src/cli.rs
+++ b/lib/src/cli.rs
@@ -194,11 +194,11 @@ pub(crate) enum InstallOpts {
     EnsureCompletion {
         /// When provided, we assume that we're being invoked from our own
         /// ostree-ext codebase.
-        #[clap(long)]
+        #[clap(long, requires = "stateroot")]
         sysroot: Option<Utf8PathBuf>,
 
         /// Must be set if sysroot is set
-        #[clap(long)]
+        #[clap(long, requires = "sysroot")]
         stateroot: Option<String>,
     },
     /// Output JSON to stdout that contains the merged installation configuration
```